packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -415,21 +432,42 @@ export class AppMainContainer extends Component<
     this.sendClearFilter();
   }
 
-  handleDataChange(data: AppDashboardData): void {
+  handleDataChange(data: DashboardData): void {
     const { updateWorkspaceData } = this.props;
 
     // Only save the data that is serializable/we want to persist to the workspace
-    const { closed, filterSets, links } = data;
-    updateWorkspaceData({ closed, filterSets, links });
+    const { closed } = data;
+    updateWorkspaceData({ closed });
   }
 
   handleGoldenLayoutChange(goldenLayout: GoldenLayout): void {
     this.goldenLayout = goldenLayout;
-  }
-
-  handleLayoutConfigChange(layoutConfig?: DashboardLayoutConfig): void {
-    const { updateWorkspaceData } = this.props;
-    updateWorkspaceData({ layoutConfig });
+    this.goldenLayout.eventHub.on(
```

Comment:
As mentioned in other review, let's have this defined as `emitDashboardOpenEvent` and `listenDashboardOpenEvent` methods that can enforce the typing.